tools/dependencies-report/src/main/resources/notices/org.eclipse.core.commands-3.6.0-NOTICE.txt

```diff
@@ -0,0 +1,88 @@
+Copyright © Eclipse Foundation, Inc. All Rights Reserved.
```

Comment:
this is the copyright of the license itself, not of the project.

In my PR that duplicated this effort #9549, I was able to dig up a project-level notice from a comment in the JAR's plugin.properties file:

https://github.com/elastic/logstash/pull/9549/files#diff-e6cb73b8ccc79daf456436815b2abdee